Criação do relatório de padrões da comunidade do github

PauloVictorFS opened the pull request:
## Descrição 
Criação do arquivo de padrões da comunidade do github de acordo com o site de referência:
 https://opensource.guide/how-to-contribute/ 
## Issues
#50 

## Tarefas gerais realizadas
* Criação do tópico de contribuição para softwares open source (como contribuir e por que contribuir)
* Criação do tópico Critérios para a criação de um software de open source (divisão organizacional, documentação e ferramentas)
* Criação do tópico de  contribuição da comunidade (como tornar atrativo para a comunidade contribuir e como contribuir)
* Criação do tópico de abertura de uma issue ( necessidades anteriores a abertura e recomendações na abertura)
* Criação do topico de abertura de um pull request (necessidades anteriores a abertura e como abrir um pull request)
* transformação do arquivo em um arquivo de contribuição e a utilização do resto do texto para o readme

PauloVictorFS committed:
criando o relatório de padrões da comunidade

criando os guias para criação de um software open source, assim como os guias de como contribuir , necessidades, entre outros

PauloVictorFS committed:
atualizando o arquivo de comunity_std

atualizei a formatação e o tamanho de algumas palavras

PauloVictorFS committed:
atualizando comunity_std

corrigindo detalhes ortográficos

dougAlvs reviewed (commented):
O guia de contribuição está bom, mas deixa o escopo dele mais específico para o nosso projeto e põe ele na raiz do repositório com o nome de "CONTRIBUTING.md".

PauloVictorFS committed:
transformando o comunity_stds em contributing.md

transformei o arquivo de comunity_stds em um arquivo de guia de contribuição , e depois reutilizarei parte da escrita para o readme

PauloVictorFS committed:
mudança de local do arquivo contributing

mudei o arquivo de contribuição para a raiz do repositório

PauloVictorFS commented:
Realizei as mudanças, por favor dê uma olhada

dougAlvs reviewed (approved):
Agora o guia de contribuição está dentro dos padrões esperados.
